FrontEnd/src/components/CheckBoxCustom.scss

```diff
@@ -0,0 +1,61 @@
+.check-box {
```

Comment:
hmm, can you add that scss extension and format with that please?
(to save us from having doing things in two separate ways)